### ISSUE TITLE
افزودن گزینه Date format

### ISSUE THREAD

**Issue by websurfer93** [karfekr/obsidian-persian-calendar#10]
بنظرم اگر یه گزینه Date format اضافه بشه خیلی مفید هست مثل چیزی که در کورپلاگین دیلی نوت هست
![image](https://github.com/maleknejad/obsidian-persian-calendar/assets/34800933/2b43bc96-9533-4019-ba2f-43eb49ba0446)
بیشتر هم نه بخاطر تغییر فرمت تاریخ، بلکه مثلا میشه بر اساس تعریف فرمت خاصی مثل این
`YYYY/MM-MMMM/YYYY-MM-DD`
میشه اتوماتیک برای ماه ها و سال ها فولدر جدا ساخت

**Comment by maleknejad** [comment]
یه بار باید ریفکتور کنم کد رو. این ایشو رو میگذارم برای اون موقع.

**Comment by pourmand1376** [comment]
این ویژگی برای من هم خیلی مهمه. اگر بشه اضافه‌اش کرد خیلی ممنون میشم.

**Comment by Arianhf** [comment]
اگه بشه این اضافه بشه واقعا عالی میشه!

**Comment by Morteza-NW** [comment]
yeah, that could fix most of my problems, hope to see this feature soon

**Comment by NedaMani** [comment]
سلام بزرگوار، امکان مسیردهی داینامیک جلالی اضافه شده، برای اضافه کردن مسیردهی میلادی باید اول [این مورد](https://github.com/karfekr/obsidian-persian-calendar/issues/53) توسعه داده بشه. 
درباره‌ی مسیردهی جلالی‌ای که اضافه شده میتونید در فایل [Readme ](https://github.com/karfekr/obsidian-persian-calendar#%D8%B1%D8%A7%D9%87%D9%86%D9%85%D8%A7%DB%8C-%D8%A7%D8%B3%D8%AA%D9%81%D8%A7%D8%AF%D9%87)همین ریپوزیتوری بخونید🙏

**Comment by NedaMani** [comment]
امکان مسیردهی داینامیک میلادی و همینطور نام‌گذاری روزنوشت به فرمت دلخواه و تنظیم هفته‌نوشت روی ابتدای سال میلادی یا شمسی در ورژن 4.8.0 پیاده‌سازی شد.
